tests/tests.py

```diff
@@ -0,0 +1,53 @@
+"""
+Simple test runner for felt-python tests.
```

Comment:
These are awesome. A couple of questions/comments that come to mind:

- Should we delete maps created for e2e tests?
- Some of the e2e tests rely on external services like a WMS service to set up a source. These might be flaky, and that might be okay, but we should keep an eye out

Reply:
- I set it up to keep the maps so they can be viewed and checked if we wanted
- Yeah, I don't think the tests will run that often, so that seems like a reasonable risk